adapter/repository/in-memory-simple/src/main/java/org/mobilitydata/gtfsvalidator/db/InMemoryGtfsDataRepository.java

```diff
@@ -422,6 +422,19 @@ public Transfer getTransferByStopPair(final String fromStopId, final String toSt
         return transferPerStopPair.get(fromStopId).get(toStopId);
     }
 
+    /**
+     * Return an immutable collection of {@code Transfer} objects representing all the rows from transfers.txt
+     *
+     * @return an immutable collection of {@link Transfer} objects representing all the rows from transfers.txt
+     */
+    @Override
+    public Collection<Transfer> getTransferAll() {
+        final Set<String> fromStopIdList = transferPerStopPair.keySet();
```

Comment:
Why not just return an unmodifiable `Set` here? We're still instantiating an entirely new `List` data structure here which is taking up space.

Reply:
Sorry, this should have read "unmodifiable `Map`"